schemas/oic.links.properties.core-schema.json

```diff
@@ -84,6 +84,26 @@
                   }
               }
           }
+      },
```

Comment:
General comment: Semantic Tags are defined in the CR as both Link Parameters and Common Properties.  This PR handles the Link Parameters but not the Common Properties.  In other words, if these show up as Resource Properties as required by the CR, then schema validation will fail.

Reply:
Don't intend on updating 120+ swagger definitions to add new common properties, it's not really a scalable way of handling things that can be used by everything.  I'd rather have the CTT check the IDD and make use of the oic.common.properties.core-schema file for "allowed ones are in here" for future stuff.